common/views/components/WobblyEdge/WobblyEdge.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+type Props = {|
+  background: string,
+  intensity?: number,
+  points?: number
+|}
+
+const WobblyEdge = ({ intensity, points, background }: Props) => (
+  <div
+    className={`wobbly-edge wobbly-edge--${background} js-wobbly-edge`}
+    data-max-intensity={intensity}
+    data-number-of-points={points}>
```

Comment:
Can we add an `isValley: boolean` prop and an associated `data-is-valley` attribute?

Reply:
yip - done.